src/main/kotlin/org/malachite/estella/commons/models/people/User.kt

```diff
@@ -11,7 +11,7 @@ data class User(
     val firstName: String, val lastName: String,
     @Column(unique = true) val mail: String
 ) {
-    public constructor(id: Int?, firstName: String, lastName: String, mail: String, password: String?) : this(
+    constructor(id: Int?, firstName: String, lastName: String, mail: String, password: String?) : this(
```

Comment:
Hmm czy przez brak public coś się nie zepsuje przypadkiem?

Reply:
nie, intelka zaznaczała na szaro, wszystko działa. nie wiem, dlaczego miałoby się zepsuć, skoro pewnie domyślnie jest publiczne, tak samo jak funkcje

Reply:
Ok myślałem że zapożyczyła to z Javy że domyślnie nie jest public